tx/textual/valuerenderer/repeated.go

```diff
@@ -0,0 +1,82 @@
+package valuerenderer
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type repeatedValueRenderer struct {
+	tr      *Textual
+	msgDesc protoreflect.MessageDescriptor
+	field   string
+	kind    string
```

Comment:
```suggestion
	fd protoreflect.FieldDescriptor
```

Reply:
Yeah, I was really trying to avoid taking the fd in.. but it appears to be unavoidable.

Reply:
resolving this since repeated has been refactored out